.eslintrc.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
how much do we want to stick to lit's eslint config? I have a bunch of rules I'm happy to recommend

Reply:
https://github.com/daniellacosse/only-web/blob/main/.eslintrc.cjs

Reply:
I'd rather stick to Google's standards.